shuffle/move/examples/main/sources/MessageTests.move

```diff
@@ -0,0 +1,23 @@
+#[test_only]
+module Sender::MessageTests {
+    use Sender::Message;
+    use Std::Signer;
+    use Std::UnitTest;
+    use Std::Vector;
+
+    fun get_account(): signer {
+        Vector::pop_back(&mut UnitTest::create_signers_for_testing(1))
+    }
+
+    #[test]
+    public(script) fun sender_can_set_message() {
+        let account = get_account();
+        Message::set_message_internal(&account,  b"Hello Blockchain");
```

Comment:
I'll put in what you could do in this PR so that you don't need to have `set_message_internal` 

```suggestion
        Message::set_message_internal(get_account(),  b"Hello Blockchain");
```